pkg/sql/codegen/xgboost/template_explain.go

```diff
@@ -22,22 +22,22 @@ type explainFiller struct {
 	DatasetSQL           string
 	ShapSummaryParams    string
 	FeatureFieldMetaJSON string
-	LabelName            string
+	LabelJSON            string
```

Comment:
What is the relationship between label and JSON? Is that XGBoost requires JSON-encoded inputs?

Reply:
It's just used to pass `ir.FieldDesc` to a Python dict object using JSON as the serialization method.